git/git_remotes.md

```diff
@@ -1,22 +1,70 @@
+
+## Assignment
+1. Read an [intro to remote repositories](https://www.git-tower.com/learn/git/ebook/en/command-line/remote-repositories/introduction#start)
+2. Take a few minutes to watch [this overview](https://www.youtube.com/watch?v=kd4jMl_3LQE) of Git remotes and remote repositories
+
+# Forking and Cloning a Remote Repository
+This section is going to cover forking and cloning Git repositories. 
 
-* Learning outcome 1
-* Learning outcome 2
-* Learning outcome 3
+Forking is a way for developers to make their own copies of a GitHub repository. It's important to note that forking is specific to the GitHub website, so it is not a part of Git. When you create a fork on GitHub, you've made an identical copy of a repository under your own user account. This allows developers to copy and change a project without affecting the original.
+
+Cloning is Git's way of getting a copy of a project. You'll have to clone a repository before you can make changes to it or see the source code on your own computer. Cloning helps you edit a repository on your local computer without making irreversible changes.
+
+Remember forking is specific to GitHub, and cloning is a feature of Git. 
+
+## Assignment
+1. Watch [episode 1.3](https://www.youtube.com/watch?v=_NrSWLQsDL4&list=PLRqwX-V7Uu6ZF9C0YMKuns9sLDzK6zoiV&index=3) of [The Coding Train](https://www.youtube.com/channel/UCvjgXvBlbQiydffZU7m1_aw)'s [Git and GitHub for Poets](https://www.youtube.com/playlist?list=PLRqwX-V7Uu6ZF9C0YMKuns9sLDzK6zoiV) to learn about making forks on GitHub
+2. Watch [episode 1.6](https://youtu.be/yXT1ElMEkW8?list=PLRqwX-V7Uu6ZF9C0YMKuns9sLDzK6zoiV) of [The Coding Train](https://www.youtube.com/channel/UCvjgXvBlbQiydffZU7m1_aw)'s [Git and GitHub for Poets](https://www.youtube.com/playlist?list=PLRqwX-V7Uu6ZF9C0YMKuns9sLDzK6zoiV) to understand cloning a remote repository
+
+# Using the git remote Command
+Here, we will discuss using the `git remote` command. First we will learn to add a new remote, then we will see how to view all remotes on a git repository. 
 
 ## Assignment
-The list of resources the user will go through to learn about the topic of this lesson. Have no more than 5 resources, ideally no more than three.
+1. Read [GitHub](https://www.github.com)'s guide for [Adding a remote](https://help.github.com/articles/adding-a-remote/)
+2. Read [Listing Remote Branches](http://gitready.com/intermediate/2009/02/13/list-remote-branches.html) from [Nick Quaranto](http://gitready.com/)
+3. Read [GitHub](https://www.github.com)'s guide for [Removing a remote](https://help.github.com/articles/removing-a-remote/)
+
+# Working with a Remote Repository
+Finally, we are going to look at making changes to a repository and making a pull request to a remote repository that you don't own.
```

Comment:
Should we fill this section out a bit more by briefly explaining what push and pull means and let the resources do the heavy lifting of going into more detail?